### PR TITLE
Docs: get started updates

### PR DESCRIPTION
**Pull request by chri2547**
This PR adjust getting started topic titles and file names to be more accurate.

### CODE REVIEW

File: docs/sources/getting-started/_index.md [hunk 1]
```diff
@@ -1,13 +1,11 @@
 +++
 aliases = ["/docs/grafana/latest/getting-started/", "/docs/grafana/latest/guides/what-is-grafana/"]
 title = "Getting started"
-weight = 10
+weight = 25
 +++
 
 # Getting started
```
Comment (brendamuir) [inline docs/sources/getting-started/_index.md:7]:
```suggestion
# Getting started
```

Comment (brendamuir) [inline docs/sources/getting-started/_index.md:7]: Should we change this to "Get started" given the discussion about not using gerunds?

File: docs/sources/getting-started/add-preconfigured-dashboard.md [hunk 2]
```diff
@@ -1,33 +1,32 @@
...
 
-# Getting started with Grafana
+# Add a preconfigured dashboard
 
 This topic helps you get started with Grafana and build your first dashboard. To learn more about Grafana, refer to [What is Grafana?]({{< relref "_index.md" >}}).
 
 > **Note:** Grafana also offers a [free account with Grafana Cloud](https://grafana.com/signup/cloud/connect-account?pg=gsdocs) to help getting started even easier and faster. You can install Grafana to self-host or get a free Grafana Cloud account.
 
-## Step 1: Install Grafana
+## Install Grafana
```
Comment (brendamuir) [inline docs/sources/getting-started/add-preconfigured-dashboard.md:14]: I think we should talk about creating a template - I read your comments in mine - and think we could come to a consensus for consistency so these docs have a similar look and feel.

File: docs/sources/getting-started/_index.md (hunk 1 above)
Comment (chri2547) [reply docs/sources/getting-started/_index.md]: Good call, made that change.

**Review by brendamuir — COMMENTED** [review]
Looks good!

**Review by brendamuir — APPROVED** [review]
Looks good!

File: docs/sources/getting-started/add-preconfigured-dashboard.md [hunk 3]
```diff
@@ -1,33 +1,32 @@
...
-description = "Guide for getting started with Grafana"
-keywords = ["grafana", "intro", "guide", "started"]
-title = "With Grafana"
+description = "Learn how to get started with Grafana by adding a preconfigured dashboard."
+title = "Add a preconfigured dashboard"
 weight = 200
 +++
 
-# Getting started with Grafana
+# Add a preconfigured dashboard
```
Comment (torkelo) [inline docs/sources/getting-started/add-preconfigured-dashboard.md:8]: I do not understand this title at all, this guide describes how to create new dashboard, add a new panel with a test data source (I SUPER stupid pointless guide). 

This guide describes nothing about importing an existing dashboard (preconfigured). It's a bit pointless to be honest.

Comment (achatterjee-grafana) [reply docs/sources/getting-started/add-preconfigured-dashboard.md:8]: > I do not understand this title at all, this guide describes how to create new dashboard, add a new panel with a test data source (I SUPER stupid pointless guide).
> 
> This guide describes nothing about importing an existing dashboard (preconfigured). It's a bit pointless to be honest.

Plus 1

File: docs/sources/getting-started/build-influxdb-dashboard.md [hunk 4]
```diff
@@ -58,7 +57,7 @@ InfluxDB publishes guidance for connecting different versions of their product t
 - If the text name of the organization or bucket doesn't work, then try the ID number.
 - If you change your bucket name in InfluxDB, then you must also change it in Grafana and your Telegraf .conf file as well.
 
-## Step 6. Add a query
+## Add a query
```
Comment (torkelo) [inline docs/sources/getting-started/build-influxdb-dashboard.md:60]: would switch these to h3 for readability

File: docs/sources/getting-started/add-preconfigured-dashboard.md (hunk 3 above)
Comment (torkelo) [reply docs/sources/getting-started/add-preconfigured-dashboard.md:8]: But it is a good guide as it describes how to first access and login to grafana (so linked to from install pages I see)

File: docs/sources/getting-started/build-influxdb-dashboard.md [hunk 5]
```diff
@@ -1,30 +1,29 @@
...
-description = "Guide for getting started with Grafana and InfluxDB"
-keywords = ["grafana", "intro", "guide", "started"]
-title = "With Grafana and InfluxDB"
+description = "Learn how to build your first InfluxDB dashboard in Grafana."
+title = "Build an InfluxDB dashboard"
 weight = 250
 +++
 
-# Getting started with Grafana and InfluxDB
+# Build an InfluxDB dashboard
```
Comment (torkelo) [inline docs/sources/getting-started/build-influxdb-dashboard.md:8]: I think the original titles where good these are much more than just building a influxdb dashboard

File: docs/sources/getting-started/build-influxdb-dashboard.md (hunk 4 above)
Comment (chri2547) [reply docs/sources/getting-started/build-influxdb-dashboard.md:60]: Made them H4s, see how you like it.

**Review by torkelo — APPROVED** [review]

**Pull request merged**